docs/setup.rst

```diff
@@ -1,32 +1,29 @@
+=====
 Setup
 =====
 
-Need to set up Conda environment first and foremost
-
-If on workstation:
-module load conda
+Set up Conda environment
+========================
 
-Create new Conda environment
-conda create -n [environmentName]
+various options work quite well:
 
-Append necessary channels
-conda config --append channels noaa-gfdl
-conda config --append channels conda-forge
 
-Run conda install on needed dependencies
-conda install noaa-gfdl::fre-cli should install the CLI package located at https://anaconda.org/NOAA-GFDL/fre-cli created from the meta.yaml file
+If on workstation
+-----------------
 
-All other dependencies used by the tools are installed along with this install (configured inside the meta.yaml), with the exception of local modules
-setup.py file allows fre.py to be ran with fre as the entry point on the command line instead of python fre.py
+``module load fre/canopy``
 
-Enter commands and follow --help messages for guidance (brief rundown of commands also provided below)
 
-If the user just runs fre, it will list all the command groups following fre, such as run, make, pp, etc. and once the user specifies a command group, the list of available subcommands for that group will be shown
+Create new Conda environment
+----------------------------
```

Comment:
Do we still have to `module load miniforge` on ws?

Reply:
Also brings up, include set up on gaea? Modules to load are different for conda

Reply:
this PR is more for structure than content- i'd trust you to include the setup on gaea more than myself

Reply:
added in the `minforge` line if one is at GFDL